src/WooCommerce.php

```diff
@@ -439,6 +439,11 @@ public static function getOrderDetailsHtmlDataAttr($order_id) {
       $order_details['coupon'] = implode(' | ', $coupons);
     }
 
+    $order_details['order_count'] = 1;
```

Comment:
1. This code here appears to generate data from an order. The order count is not really part of an order, but instead of the user / customer. Is there a method generating data about the current user/customer, where you could move this instead?

1. The default count needs to be zero, as not every authenticated user on the site needlessly has ordered something already.

Reply:
1. The code generates the data from an order if available, otherwise it defaults to 1, since this is the customer's first order, and saves to LocalStorage. Next time on order is created from the same browser, LocalStorage order_count will be incremented and will overwrite the value coming from php. This way we will have more accurate tracking for the existing customers, instead of always simply starting with order_count = 1.

2. If no previous order_count is available in LocalStorage or php, the default value is 1, not zero, since this is the customer's first order, so order_count = 1.

Reply:
1. These considerations should be documented in code, as they are not obvious.

Reply:
I just did, please review.